src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/MethodologyVersionViewModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿#nullable enable
+using System;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+public record MethodologyVersionViewModel
+{
+    public Guid Id { get; set; }
+
+    public bool Amendment { get; set; }
+
+    // TODO EES-3596 Remove InternalReleaseNote added to support the old Admin dashboard
```

Comment:
Would probably drop this comment as we should probably keep this in. As we don't display it in the new dashboard, we should at least show it somewhere. This would probably be best shown in a methodology's 'Sign off' section's summary list.

Reply:
The existing and new dashboard now both use the new `GET /api/publication/{publicationId}/methodologies` added by this PR which returns the new `MethodologyVersionViewModel` and maps to `MethodologyVersion` on the front end (renamed from `MethodologyVersionListItem` just now). This has this temporary `InternalReleaseNote` to keep the existing dashboard working, but not displayed on the new dashboard, hence the TODO to remove it in future.

The Methodology page / 'Sign off' tab are using `GET /api/methodology/{methodologyVersionId}/summary` which returns `MethodologyVersionSummaryViewModel` and maps to `BasicMethodologyVersion` . This hasn't changed but already has a `LatestInternalReleaseNote`. Since we weren't ready to display multiple approval histories yet for methodologies this should probably have been left as `InternalReleaseNote` for now rather than 'latest'`. 

I've just added 3ce3eb8e3da7c94290e05490b36e9e0f3ca320dd to display it on the 'Sign off' tab.

It does seem a bit strange that the variant returned for the content page is the 'Summary' view model.

Reply:
Added 850ffec809e9829f7ce36a0e0a6f17dfc85c708c to display it on the 'Sign off' tab.

The existing and new dashboard now both use the new `GET /api/publication/{publicationId}/methodologies` added by this PR which was returning a new `MethodologyVersionViewModel` which mapped to `MethodologyVersion` on the front end.

In 227af3491faa80974e172b20ff169334bd6e3bdc I've done a big switcharoo as discussed, renaming as follows:

`MethodologyVersionSummaryViewModel` -> `MethodologyVersionViewModel` (BE) with corresponding `BasicMethodologyVersion` -> `MethodologyVersion` (FE), extending from new `BaseMethodologyVersion`.

`MethodologyVersionViewModel` -> `MethodologyVersionSummaryViewModel` (BE) with corresponding `MethodologyVersion` -> `MethodologyVersionSummary` (FE), also extending from the new `BaseMethodologyVersion`.

`LatestInternalReleaseNote` was already on the view model used by the content page, which is now displayed on the 'Sign off' tab. Since we aren't ready to display multiple approval histories yet for methodologies I've renamed if from `LatestInternalReleaseNote` to `InternalReleaseNote`.

The TODO on `MethodologyVersionSummaryViewModel` has been left as we can remove it in future once the old dashboard is removed.




It does seem a bit strange that the variant returned for the content page is the 'Summary' view model.